src/components/zones/MainGroupList.tsx

```diff
@@ -74,93 +69,136 @@ export default function MainGroupList() {
 
 let timeout: ReturnType<typeof setTimeout>;
 // eslint-disable-next-line react/display-name
-const UnfolTreeWrapper = React.memo(
-  ({
-    firstChildrenTree,
-    zones,
-    openedZoneIds,
-  }: {
-    firstChildrenTree: TreeZoneEl[] | undefined;
-    zones: Zone[];
-    openedZoneIds: string[];
-  }) => {
-    const isNewImportSvgHook = useIsNewImportedSvg();
-    const [displayContent, setDisplayContent] = useState(false);
-    const [showSpinner, setShowSpinner] = useState(false);
-    // when isNewImportSvg change from true to false, wait 300ms before rendering the component (big SVG can take long to render the tree)
-    useEffect(() => {
-      if (isNewImportSvgHook) {
-        setShowSpinner(false);
-        return setDisplayContent(false);
-      }
-
-      setShowSpinner(true);
-      if (timeout) clearTimeout(timeout);
-      timeout = setTimeout(() => {
-        setDisplayContent(true);
-        setShowSpinner(false);
-      }, 300);
+const UnfoldedTreeWrapper = React.memo(() => {
+  const zones = useAppSelector(
+    (store) => store.zonesSlice.zones.filter((z) => z.createdFrom === "figma"),
+    isEqual
+  );
+  const firstChildrenTree = useAppSelector(
+    (store) => store.zonesSlice.tree?.[0]?.children,
+    isEqual
+  );
+  const openedZoneIds = useAppSelector(getSelectedFigmaZoneIds);
 
-      return () => {
-        if (timeout) clearTimeout(timeout);
-      };
-    }, [isNewImportSvgHook]);
+  const isNewImportSvgHook = useIsNewImportedImage();
```

Comment:
should it be `isNewImportImageHook` since not necessary SVG?

Reply:
it's preferred to name hooks with the use prefix
but we can still rename it if you want

Reply:
yes but I meant to rename the const cariable `isNewSvg ` to `isNewImage` not the hook

Reply:
ah sorry yes np